frontend/src/store/wallet.ts

```diff
@@ -22,6 +22,7 @@ import { ERC20_ABI, MAINNET_PROVIDER, MULTICALL_ABI, MULTICALL_ADDRESS } from 's
 import { BigNumber, Contract, ExternalProvider, Web3Provider, parseUnits } from 'src/utils/ethers';
```

Comment:
_Random placement for comment_

This is actually great, thanks for picking this up. 

One nit is, could we put in a placeholder for the Token logo and Amount when API is not able to fetch those details on the receive table? And maybe handle 'LogoURI' errors a bit better?
<img width="1066" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/b996f115-fd58-4617-b056-90bedfadb831">

Reply:
Thanks Gary! Great catch.. fixed: [ddf1367](https://github.com/ScopeLift/umbra-protocol/pull/647/commits/ddf136714f66f9f238f2c2239f4013ce75baa7b0)

Reply:
Looks good now 
<img width="846" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/8b25a519-2910-4410-898c-1928905943bb">